vnext/ReactUWP/Views/PickerViewManager.cpp

```diff
@@ -35,13 +35,16 @@ class PickerShadowNode : public ShadowNodeBase
   PickerShadowNode();
   void createView() override;
   void updateProperties(const folly::dynamic&& props) override;
+  bool IsExternalLayoutDirty() const override { return m_newItems; }
+  void DoExtraLayoutPrep(YGNodeRef yogaNode) override;
 
 private:
   void RepopulateItems();
   static void OnSelectionChanged(IReactInstance& instance, int64_t tag, folly::dynamic&& value, int32_t selectedIndex, folly::dynamic&& text);
 
   folly::dynamic m_items;
   int32_t m_selectedIndex = -1;
+  bool m_newItems = false;
```

Comment:
I'd like to see m_newItems be something like m_hasNewItems so it doesn't sound like a collection of the items.

Reply:
Yes, will do.